app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -453,6 +453,49 @@ def test_can_review_false_for_non_review_publish_status(self, publish_status):
 
         self.assertFalse(experiment.can_review(UserFactory.create()))
 
+    @parameterized.expand(
+        [
+            NimbusExperiment.Status.DRAFT,
+            NimbusExperiment.Status.LIVE,
+        ]
+    )
+    def test_timeout_changelog_for_timedout_publish_flow(self, experiment_status):
```

Comment:
I *think* all the case we wanted to cover here are within DRAFT and LIVE experiment statuses